pkg/controller/s3/bucket_test.go

```diff
@@ -117,6 +117,9 @@ func TestObserve(t *testing.T) {
 				result: managed.ExternalObservation{
 					ResourceExists:   true,
 					ResourceUpToDate: true,
+					ConnectionDetails: map[string][]byte{
+						corev1alpha1.ResourceCredentialsSecretEndpointKey: []byte(s3Testing.BucketName),
+					},
```

Comment:
Looks like you need to add the region key here also :)

Reply:
Thanks, my bad! :smile: